client/my-sites/people/delete-user/index.jsx

```diff
@@ -18,7 +18,7 @@ const Card = require( 'components/card' ),
 	FormButtonsBar = require( 'components/forms/form-buttons-bar' ),
 	AuthorSelector = require( 'blocks/author-selector' ),
 	UsersActions = require( 'lib/users/actions' ),
-	Gravatar = require( 'components/gravatar' ),
+	Gravatar = require( 'components/gravatar' ).default,
```

Comment:
Let's add an Gravatar import after the requires. We can go back to clean up the rest of the requires in a follow up janitorial PR